src/Directory.Build.targets

```diff
@@ -3,6 +3,7 @@
     <!-- Reset these properties back to blank, since they are defaulted by Microsoft.NET.Sdk -->
     <FileAlignment Condition="'$(FileAlignment)' == '512'"></FileAlignment>
     <ErrorReport Condition="'$(ErrorReport)' == 'prompt'"></ErrorReport>
+    <NETCoreAppMaximumVersion>5.0</NETCoreAppMaximumVersion>
```

Comment:
Again, why is this necessary?

Reply:
replied above

Reply:
We should probably not have it defined in two places though, cand we just have it here and have restore directory.build.props import this?

Reply:
yes I moved it to the directory.build.targets in the root. However we will remove when sdk adds netcoreapp5 tfm. 
We cant add it directory.Build.Props because when the projects include Microsoft.net.sdk, they overwrite the property.